contact/pubspec.lock

```diff
@@ -992,6 +992,15 @@ packages:
       url: "https://pub.dev"
     source: hosted
     version: "3.2.1"
+  sanitize_html:
```

Comment:
Comment on why we need to fork this at the minimum

Reply:
Ideally we shall also have an upstream ticket / PR so that we eventually get rid of this fork!

Reply:
Sanitize_html is restricting `Tags` and `Attributes`. So some of our own tags and attributes (`signature, public asset`) will be lost when sanitizing html.

Reply:
Add it in a comment, along with upstream PR

Reply:
done